### ISSUE TITLE
Call to undefined function config_del()

### ISSUE THREAD

**Issue by lolandese** [backdrop-contrib/date#5]
As title. Occurring on uninstall of date_api. Likely due to a bulk Find & Replace of _variable_ with _config_.

``` php
diff --git a/date_api/date_api.install b/date_api/date_api.install
index 39dfd62..1b2677b 100644
--- a/date_api/date_api.install
+++ b/date_api/date_api.install
@@ -98,7 +98,7 @@ function date_api_uninstall() {
     'date_api_use_iso8601',
   );
   foreach ($variables as $variable) {
-    config_del($variable);
+    variable_del($variable);
   }

   if (db_table_exists('views_display')) {
```

Other occurrences:

Searching 106 files for "config_del"

/home/martin/www/backdrop/modules/date/date_views/date_views.install:
   22   */
   23  function date_views_uninstall() {
   24:   config_del('date_views_month_format_with_year');
   25:   config_del('date_views_month_format_without_year');
   26:   config_del('date_views_day_format_with_year');
   27:   config_del('date_views_day_format_without_year');
   28:   config_del('date_views_week_format_with_year');
   29:   config_del('date_views_week_format_without_year');
   30  }
   31  

/home/martin/www/backdrop/modules/date/tests/date_api.test:
  425     */
  426    public function tearDown() {
  427:     config_del('date_first_day');
  428:     config_del('date_api_use_iso8601');
  429      parent::tearDown();
  430    }

8 matches across 2 files

**Comment by lolandese** [comment]
Better still, remove these all together and test with compatibility mode turned off in _settings.php_.

**Comment by Graham-72** [comment]
Hopefully dealt with in commit https://github.com/backdrop-contrib/date/commit/42100ca0e742e91c40417588c2708707c171ad44

**Comment by lolandese** [comment]
What about the other occurrences?

Searching 106 files for "config_del"

/home/martin/www/backdrop/modules/date/date_views/date_views.install:
22 */
23 function date_views_uninstall() {
24: config_del('date_views_month_format_with_year');
25: config_del('date_views_month_format_without_year');
26: config_del('date_views_day_format_with_year');
27: config_del('date_views_day_format_without_year');
28: config_del('date_views_week_format_with_year');
29: config_del('date_views_week_format_without_year');
30 }
31

/home/martin/www/backdrop/modules/date/tests/date_api.test:
425 */
426 public function tearDown() {
427: config_del('date_first_day');
428: config_del('date_api_use_iso8601');
429 parent::tearDown();
430 }

8 matches across 2 files

**Comment by Graham-72** [comment]
Yes, I have yet to revise date_views and tests. I will do so very soon.
